node/src/actors/peers_manager/mod.rs

```diff
@@ -112,6 +112,18 @@ impl PeersManager {
             _ => {}
         }
     }
+
+    /// Method to try peers periodically to move peers from new to tried
+    pub fn feeler(&mut self, ctx: &mut Context<Self>, feeler_peers_period: Duration) {
+        // Schedule the discovery_peers with a given period
+        ctx.run_later(feeler_peers_period, move |act, ctx| {
+            if let Some((key, peer)) = act.peers.get_new_random() {
+                act.peers.remove_from_new_with_index(&[key]);
```

Comment:
So if the `try_peer` fails, this peer is removed from new. Is that the expected behavior?

Reply:
if try_peer fails means that it is not a useful peer